cisco_sdwan/assets/monitors/device_unreachable.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 2,
+  "created_at": "2024-04-05",
+  "last_updated_at": "2024-04-05",
+  "title": "Cisco SD-WAN Device is unreachable",
+  "tags": [
+    "integration:cisco-sdwan"
+  ],
+  "description": "This monitor checks the reachability of Cisco SD-WAN devices within a specified namespace. It triggers an alert if a device becomes unreachable, indicating potential network connectivity issues.",
+  "definition": {
+	"name": "[Cisco SD-WAN] Device unreachable alert on {{hostname.name}} in namespace {{device_namespace.name}}",
+	"type": "query alert",
+	"query": "avg(last_5m):max:cisco_sdwan.device.reachable{*} by {hostname,system_ip,device_namespace} < 0.8",
+	"message": "{{#is_alert}}\nA network device {{hostname.name}} with IP {{system_ip.name}} in namespace {{device_namespace.name}} is unreachable.\n{{/is_alert}}\n{{#is_alert_recovery}}\nA network device {{hostname.name}} with IP {{system_ip.name}} in namespace {{device_namespace.name}} is reachable again.\n{{/is_alert_recovery}}\n\nTo know more about the status of your device, you can have more information from the [NDM page for the device {{device_namespace.name}}:{{system_ip.name}}](/infrastructure/devices/graph?inspectedDevice={{device_namespace.name}}%3A{{system_ip.name}}).",
+	"tags": [],
+	"options": {
+		"thresholds": {
+			"critical": 0.8
+		},
+		"notify_audit": false,
+		"include_tags": true,
+		"new_group_delay": 60,
+		"notify_no_data": false,
+		"silenced": {},
+		"avalanche_window": 10
```

Comment:
I'm curious, why does this one have more json fields than the previous two?

Reply:
good catch, don't know what happened there
Cleaned this in 794cd66